stable/postgresql/Chart.yaml

```
@@ -1,7 +1,7 @@
 apiVersion: v1
 name: postgresql
-version: 6.3.7
-appVersion: 11.5.0
+version: 6.3.8
+appVersion: 11.5.1
```

Why are you bumping the `appVersion`?
```suggestion
appVersion: 11.5.0
```

---

in fact i do not really know difference between appVersion and version, is it need or not?